projects/core/src/cms/store/reducers/navigation-entry-item.reducer.ts

```diff
@@ -24,9 +24,7 @@ export function reducer(
               [`${component.uid}_AbstractCMSComponent`]: component,
             };
           },
-          {
-            ...{},
-          }
```

Comment:
this code cleanup is done by the way. it has nothing to do with the main purpose (bugfix) of this PR